financial-templates-lib/logger/ConsoleTransport.js

```diff
@@ -1,22 +1,35 @@
 // This transport enables Winston logging to the console.
 const winston = require("winston");
+const { format } = winston;
+const { combine, timestamp, colorize, printf } = format;
 
-const alignedWithColorsAndTime = winston.format.combine(
-  winston.format.colorize(),
-  winston.format.timestamp(),
-  winston.format.printf(info => {
-    const { timestamp, level, ...args } = info;
+const customFormat = combine(
+  // Adds level-based coloring.
+  colorize(),
+  // Adds timestamp.
+  timestamp(),
+  printf(info => {
+    const { timestamp, level, error, ...args } = info;
 
     // This slice changes a timestamp formatting from `2020-03-25T10:50:57.168Z` -> `2020-03-25 10:50:57`
     const ts = timestamp.slice(0, 19).replace("T", " ");
-    return `${ts} [${level}]: ${Object.keys(args).length ? JSON.stringify(args, null, 2) : ""}`;
+    let log = `${ts} [${level}]: ${Object.keys(args).length ? JSON.stringify(args, null, 2) : ""}`;
```

Comment:
sad that you have to do this but it looks perfect as a fix. thanks for doing this!